docs/CHANGELOG.asciidoc

```diff
@@ -25,6 +25,52 @@ Review important information about the {kib} 8.x releases.
 
 --
 
+[[release-notes-8.2.1]]
+== {kib} 8.2.1
+
+coming::[8.2.1]
+
+The 8.2.1 release includes the following bug fixes.
+
+[float]
+[[enhancement-v8.2.1]]
+=== Enhancements
+Elastic Security::
+For the Elastic Security 8.2.1 release information, refer to {security-guide}/release-notes.html[_Elastic Security Solution Release Notes_].
+Monitoring::
+* Adds the ability collect Telemetry {kibana-pull}130498[#130498]
+* Adds the ability to report panels in dashboards by type {kibana-pull}130166[#130166]
+
+[float]
+[[fixes-v8.2.1]]
+=== Bug fixes
+Discover::
+* Fixes Document Explorer infinite height growth {kibana-pull}131723[#131723]
+* Fixes links in helper callouts {kibana-pull}130873[#130873]
+Elastic Security::
+For the Elastic Security 8.2.1 release information, refer to {security-guide}/release-notes.html[_Elastic Security Solution Release Notes_].
+Fleet::
+* Removes legacy component templates on package install {kibana-pull}130758[#130758]
+Lens & Visualizations::
+* Fixes time shift bug in *Lens* {kibana-pull}132000[#132000]
+* Removes the rule of color applying {kibana-pull}131128[#131128]
+Machine Learning::
+* Removes alerting_rules from general job list items {kibana-pull}131936[#131936]
+* Fixes management app docs links {kibana-pull}130776[#130776]
+Management::
+* Restores data view management field type conflict detail modal {kibana-pull}132197[#132197]
+* Fixes test data for import and export between versions tests {kibana-pull}131470[#131470]
+* Fixes condition auto-completion for templates {kibana-pull}126881[#126881]
```

Comment:
Can we change this to "Fixes condition auto-completion for templates in Console"?